stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -444,6 +444,10 @@ spec:
         - name: installer-info
           mountPath: "/artifactory_bootstrap/info/installer-info.json"
           subPath: installer-info.json
+      {{- if .Values.metrics.enabled }}
+        - name: artifactory-logs
+          mountPath: /opt/jfrog/artifactory/var/log
```

Comment:
Use `.Values.artifactory.persistence.mountPath` for the base path

Reply:
I think that is set to `/var/opt/jfrog/artifactory` which is slightly different and used for the artifact persistence not the log files.

https://github.com/jfrog/charts/blob/master/stable/artifactory-ha/values.yaml#L587

Reply:
/var/opt/jfrog/artifactory/log/ also has the logs. I think its better to use that. Same for the JFROG_HOME. I don't think you need anything else other than the log folder, right?

Reply:
No I don't, I'll make this change and thanks for the clarification

Reply:
👍

Reply:
Pushed up a fix